getting_started/workflow/assets/escn_exporter/animation.rst

```diff
@@ -0,0 +1,36 @@
+Animation
+=========
+
+Supported Animation
+-------------------
+ - transform animation of all types of object
+ - transform animation of pose bone
+
+
+
+Place of AnimationPlayer
+---------------------------
+Godot and Blender have different structure to store animation data.
+In Godot animation data is stored in an AnimationPlayer node, instead
+of in each node.
+
+The exporter has an option :code:`Separate AnimationPlayer For Each Object`
+controls how the exported AnimationPlayers are placed.
```

Comment:
Perhaps explain why you would want to chose separate or not. (IIRC it was that a single animation player cannot play multiple animations at the same time)